I see I missed this warning

```
bitwriter.c:335:49: warning: 'bitwriter_grow_' is static but used in inline function 'FLAC__bitwriter_write_raw_uint32_nocheck' which is not static
  335 |         if(bw->capacity <= bw->words + bits && !bitwriter_grow_(bw, bits))
      |
```

Need to check that out as well

--- a/src/libFLAC/bitwriter.c
+++ b/src/libFLAC/bitwriter.c
@@ -313,7 +313,7 @@ FLAC__bool FLAC__bitwriter_write_zeroes(FLAC__BitWriter *bw, uint32_t bits)
 	return true;
 }
 
-FLAC__bool FLAC__bitwriter_write_raw_uint32_nocheck(FLAC__BitWriter *bw, FLAC__uint32 val, uint32_t bits)
+static inline FLAC__bool FLAC__bitwriter_write_raw_uint32_nocheck(FLAC__BitWriter *bw, FLAC__uint32 val, uint32_t bits)
 {
 	register uint32_t left;
 

